samples/client/petstore/python/.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.2.3-SNAPSHOT
\ No newline at end of file
+2.3.0-SNAPSHOT
```

Comment:
> We released 2.2.3 (previous master) over the weekend and master is now 2.3.0.

If this is the case, why does this diff and many other unrelated changes appear?

Anyway, the fix by itself is correct, so if the other matters are fine, it is good to merge this in.